### ISSUE TITLE
Checkboxes and Radiobutton aren't visually differentiated when focused

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#215]
Add visual indication when a checkbox and/or radio button receives focus.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.5.5](https://github.com/blackbaud/skyux/releases/tag/v1.5.5)
